hazelcast/src/main/java/com/hazelcast/map/impl/MapManagedService.java

```diff
@@ -56,6 +56,8 @@ public void reset() {
 
     @Override
     public void shutdown(boolean terminate) {
+        mapServiceContext.getExpirationManager().stop();
+
```

Comment:
Can this throw an exception?

Reply:
Also, won't this be covered by the Lifecycle Notification on the ExpirationManager?

Reply:
>Can this throw an exception?
No according to contract, it should throw.

>Also, won't this be covered by the Lifecycle Notification on the ExpirationManager?
Calling it has no harm due to the idempotency, but you are right it is redundant, i will remove it.